ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -68,7 +68,7 @@ r, ".* ERR bgp#bgpcfgd: .*BGPVac.*attribute is supported.*"
 
 # https://msazure.visualstudio.com/One/_workitems/edit/14233938
 r, ".* ERR swss\d*#fdbsyncd: :- readData: netlink reports an error=-25 on reading a netlink socket.*"
-r, ".* ERR swss\d*#fdbsyncd: :- readData: netlink reports an error=-33 on reading a netlink socket.*"
+r, ".* ERR swss\d*#.*: :- readData: netlink reports an error=-33 on reading a netlink socket.*"
```

Comment:
Should we say .*syncd* instead just to be safe?

Reply:
GH is messing up formatting :(

Reply:
Thanks, done.